### ISSUE TITLE
Reuse scenarios

### ISSUE THREAD

**Issue by sdruskat** [sdruskat/research-software-categories#1]
Thinking about requirements engineering for research software, and experimentally adapting the categorization in the CiSE paper, I missed a categorization that expresses "usage" or "reuse". 

IMHO, there is a significant difference between categories here, which also impacts how we think about software in terms of (R)SE. E.g., the requirements for a data analysis script that is single use/single purpose software (used to produce visualization of data for a paper) is completely different from platform and infrastructure software.

There may be a general terminological and/or semantic issue with the underspecified term "software", but it's not a can of worms to open in this work. In any case, it is my experience that the RSE community includes "smaller" software such as scripts in their scope.

I feel that there could also be a procedural aspect to this category, as it can describe potential or future use as well a snapshot, rather than, e.g., the development process. But maybe that's true for all categorization categories.

In summary:

> As a user of our categorization work, I want a dimension that expresses the potential for reuse (and by extension the measure of impact on other people) that software may have by means of a categorization such as "reuse scenario", "usage potential" or similar, so that I can more fully express the properties of a software.

Note that this is different from "criticality", as in research, even the smallest single dev/single user/single purpose script may be highly critical in its effect on the research outcome, i.e., project size does not correlate with criticality.
